ovn_cluster.sh

```diff
@@ -695,10 +695,25 @@ function start-chassis() {
     start $ovn_central $ovn_remote $ovn_add_chassis
 }
 
+function os-image-pull() {
+    cmd="${RUNC_CMD} pull ${OS_IMAGE}"
+    retries=0
+    until [ $retries -ge 20 ]
```

Comment:
Nit: shall we make the retry limit configurable via an env variable?

Otherwise, the rest LGTM.

Reply:
> Nit: shall we make the retry limit configurable via an env variable?
> 
> Otherwise, the rest LGTM.

That is a good idea! Done.